core/util.go

```diff
@@ -536,3 +538,29 @@ func LoadCert(filename string) (cert *x509.Certificate, err error) {
 	cert, err = x509.ParseCertificate(block.Bytes)
 	return
 }
+
+// RetryJitter is used to prevent bunched retried queries from falling into lockstep,
+// it needs to be exported for testing
+var RetryJitter = 0.2
```

Comment:
Looks like this became a var and was exported in 2461726d9dff8bed2d38d51c6b523e993a09ec23, to be able to set it to zero for testing. But that's probably not a good idea: instead, can we test for a certain range?